models/public/mask_rcnn_inception_resnet_v2_atrous_coco/mask_rcnn_inception_resnet_v2_atrous_coco.md

```diff
@@ -0,0 +1,78 @@
+# mask_rcnn_inception_resnet_v2_atrous_coco
+
+## Use Case and High-Level Description
+
+Mask R-CNN Inception Resnet V2 Atrous trained on COCO dataset. Used for object instance segmentation. For details, see [paper](https://arxiv.org/pdf/1703.06870.pdf)
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Instance segmentation                     |
+| GFlops                          | 675.314                                   |
+| MParams                         | 92.368                                    |
+| Source framework                | Tensorflow\*                              |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `image_tensor`, shape: [1x800x800x3], format: [BxHxWxC],
```

Comment:
I'm not sure about size specification, in TF model has dynamic shape and image scaled with aspect ratio saving (min size is 800, max size is 1365)